src/style.scss

```diff
@@ -1,12 +1,14 @@
-// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
-// See LICENSE in the project root for license information.
+/* Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license */
```

Comment:
Why are all these comments changed from Sass comments to CSS comments? There is no reason for this to be output into the CSS.

Reply:
<img width="1133" alt="Screen Shot 2023-06-08 at 6 24 22 PM" src="https://github.com/AlaskaAirlines/auro-button/assets/55807669/3eeebb3c-b43f-46d7-9019-abc38aaa78c7">

The comments were changed because this error was appearing in the console when running a build. The error was fixed by changing the comment type.

Reply:
I'm noticing other repos provide certain extensions and plugins to support `.scss` styling ([auro-menu](https://github.com/AlaskaAirlines/auro-menu/blob/main/.stylelintrc#L2), [auro-datepicker](https://github.com/AlaskaAirlines/auro-datepicker/blob/main/.stylelintrc#L2), [auro-hyperlink](https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/.stylelintrc#L3)), we may want to include these rules here for proper commenting. 

(this may also solve the unknown `@` rule issue??)

Reply:
The problem is that for some reason not all the dependencies are listed? In the Generator package template there are the following: 

```
    "stylelint": "",
    "stylelint-config-idiomatic-order": "",
    "stylelint-config-standard": "",
    "stylelint-config-standard-scss": "",
    "stylelint-order": "",
    "stylelint-scss": "",
```

In this project, all there is

```
    "stylelint": "^15.7.0",
    "stylelint-config-standard": "^33.0.0",
```

Update these dependencies and then also update the .stylelintrc based on the configs from the Generator. This will also require updates to the code itself to meet the new standards. 

Although, I am ok with NOT updating to use `stylelint-config-idiomatic-orde`.

Reply:
New commit was added to fix this issue